src/server/app.js

```diff
@@ -42,7 +42,7 @@ import session from 'express-session';
 
 
 // Initialize log-to-stdout  writer
-require('log-node')();
+require('log-node')(); // eslint-disable-line import/no-commonjs
```

Comment:
This too can be changed to use ES6 import syntax instead
```javascript

import logNode from 'log-node';
[...]
// Initialize log-to-stdout  writer
logNode();
```

Reply:
Done!